dev-libs/argtable/argtable2-13.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="An ANSI C command line parser"
+DESCRIPTION=" Argtable is an ANSI C library for parsing GNU style command \
+line options with a minimum of fuss. It enables a program's command line \
+syntax to be defined in the source code as an array of argtable structs. \
+The command line is then parsed according to that specification and the \
+resulting values are returned in those same structs where they are accessible \
+to the main program."
+HOMEPAGE="http://argtable.sourceforge.net/"
+COPYRIGHT="1998-2011 Stewart Heitmann"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://prdownloads.sourceforge.net/argtable/argtable2-13.tar.gz"
+CHECKSUM_SHA256="8f77e8a7ced5301af6e22f47302fdbc3b1ff41f2b83c43c77ae5ca041771ddbf"
+
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	argtable2$secondaryArchSuffix = $portVersion
+	lib:libargtable2$secondaryArchSuffix = 0.1.8 compat >= 0
```

Comment:
Can we get a `libVersion` treatment? :)

Reply:
AFAIK this is the last in this line, argtable3 is newer and would mean a rewrite of this I gues ;)